includes/feed/class-feed.php

```diff
@@ -55,7 +55,7 @@ public function get_posts_allow_side_effects() : array {
 	/**
 	 * Maybe Set Update Schedule
 	 */
-	public function maybe_set_update_schedule() : void {
+	public function maybe_set_update_schedule() {
```

Comment:
Is there a specific reason to get rid of the type hints here? `void` isn't terribly important but it's more accurate than nothing.

Reply:
Probably better to use them, added these back